### PR TITLE
Update free-programming-interactive-tutorials-en.md

### PR DESCRIPTION
**Pull request by tutlane**
Added good resources to learn c#, python, bootstrap, node.js, etc. Please check it, if it is good, keep it in the list.

## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [ ] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [ ] Put lists  in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction)

## Followup

- Check the output of Travis-CI for linter errors!

### CODE REVIEW

**Comment by SethFalco** [comment]
Hey, and welcome to the repository! :wave:

I checked Tutlane out, but these look more like books rather than interactive tutorials?
Maybe we should move them over to a books list instead? 

Also, could we use the names as given by the resource, there's no need to attribute to Tutlane at the end of each resource's title.

Could you please ensure all lists, categories, and sections or subsections remain in alphabetical order?

**Comment by tutlane** [comment]
Modified the details. Please check it and merge the details

**Comment by eshellman** [comment]
I agree with Seth that that these would fit our books category, but we usually defer to whatever the resource describes itself as, so keep this on the interactive tutorials page.
Thanks for changing the title; we place the creator name _after_ the link - check out the other similarly named resources. To distinguish the tutlane resources from identically named items, use 'tutlane' as the creator. Also, note that the title change has caused our linter to change its opinion on how to alphabetize the links, so be sure to check the travis output.

File: more/free-programming-interactive-tutorials-en.md [hunk 1]
```diff
@@ -49,6 +49,7 @@
 
 ### Android
 
+* [Android Tutorial](https://www.tutlane.com/tutorial/android)
```
Comment (davorpa) [inline more/free-programming-interactive-tutorials-en.md:52]: Add creator to disambiguate resources

```suggestion
* [Android Tutorial](https://www.tutlane.com/tutorial/android) - tutlane
```

File: more/free-programming-interactive-tutorials-en.md [hunk 2]
```diff
@@ -66,6 +67,7 @@
 ### C Sharp
 
 * [C# Tutorial](https://www.w3schools.com/cs) - W3Schools
+* [C# Tutorial](https://www.tutlane.com/tutorial/csharp)
```
Comment (davorpa) [inline more/free-programming-interactive-tutorials-en.md:70]: Add creator to disambiguate resources

```suggestion
* [C# Tutorial](https://www.tutlane.com/tutorial/csharp) - tutlane
* [C# Tutorial](https://www.w3schools.com/cs) - W3Schools
```

File: more/free-programming-interactive-tutorials-en.md [hunk 3]
```diff
@@ -153,6 +155,7 @@
 
 #### Bootstrap
 
+* [Bootstrap Tutorial](https://www.tutlane.com/tutorial/bootstrap)
```
Comment (davorpa) [inline more/free-programming-interactive-tutorials-en.md:158]: Add creator to disambiguate resources

```suggestion
* [Bootstrap Tutorial](https://www.tutlane.com/tutorial/bootstrap) - tutlane
```

File: more/free-programming-interactive-tutorials-en.md [hunk 4]
```diff
@@ -188,6 +191,7 @@
 * [Angular Basics](http://www.angularjsbook.com)
 * [Angular Tutorial](https://www.w3schools.com/angular/) - W3Schools
 * [AngularJS - Step by Logical Step](http://nicholasjohnson.com/angular-book/)
+* [AngularJS Tutorial](https://www.tutlane.com/tutorial/angularjs)
```
Comment (davorpa) [inline more/free-programming-interactive-tutorials-en.md:194]: Add creator to disambiguate resources

```suggestion
* [AngularJS Tutorial](https://www.tutlane.com/tutorial/angularjs) - tutlane
```

File: more/free-programming-interactive-tutorials-en.md [hunk 5]
```diff
@@ -241,6 +245,7 @@
 
 * [Node School](http://nodeschool.io)
 * [Node.js Tutorial](https://www.w3schools.com/nodejs) - W3Schools
+* [Node.js Tutorial](https://www.tutlane.com/tutorial/nodejs)
```
Comment (davorpa) [inline more/free-programming-interactive-tutorials-en.md:248]: Add creator to disambiguate resources

```suggestion
* [Node.js Tutorial](https://www.tutlane.com/tutorial/nodejs) - tutlane
* [Node.js Tutorial](https://www.w3schools.com/nodejs) - W3Schools
```

File: more/free-programming-interactive-tutorials-en.md [hunk 6]
```diff
@@ -281,6 +286,7 @@
 * [Python Koans](https://github.com/gregmalcolm/python_koans)
 * [Python Pandas Tutorial: A Complete Introduction for Beginners](https://www.learndatasci.com/tutorials/python-pandas-tutorial-complete-introduction-for-beginners/) - George McIntire, Brendan Martin, Lauren Washington
 * [Python Tutorial](https://www.w3schools.com/python) - W3Schools
+* [Python Tutorial](https://www.tutlane.com/tutorial/python)
```
Comment (davorpa) [inline more/free-programming-interactive-tutorials-en.md:289]: Add creator to disambiguate resources

```suggestion
* [Python Tutorial](https://www.tutlane.com/tutorial/python) - tutlane
* [Python Tutorial](https://www.w3schools.com/python) - W3Schools
```

File: more/free-programming-interactive-tutorials-en.md [hunk 7]
```diff
@@ -313,6 +319,7 @@
 * [SQL at Codecademy](https://www.codecademy.com/courses/learn-sql)
 * [SQL Tutorial](https://www.w3schools.com/sql) - W3Schools
 * [SQLBolt](http://sqlbolt.com)
+* [SQL Server Tutorial](https://www.tutlane.com/tutorial/sql-server)
```
Comment (davorpa) [inline more/free-programming-interactive-tutorials-en.md:322]: - Add creator to disambiguate resources
- Fix Travis linting (https://app.travis-ci.com/github/EbookFoundation/free-programming-books/builds/225315532#L225)

```suggestion
* [SQL at Codecademy](https://www.codecademy.com/courses/learn-sql)
* [SQL Server Tutorial](https://www.tutlane.com/tutorial/sql-server) - tutlane
* [SQL Tutorial](https://www.w3schools.com/sql) - W3Schools
* [SQLBolt](http://sqlbolt.com)
```

**Review by davorpa — CHANGES_REQUESTED** [review]
I just propose inline changes to:

- Disambiguate resources adding creator @tutlane
- Fix Travis linting (https://app.travis-ci.com/github/EbookFoundation/free-programming-books/builds/225315532#L225)

as @eshellman suggest

Please merge and check again

**Comment by eshellman** [comment]
Be sure to check the output of Travis-CI for linter errors. Alphabetical order and spacing are the most common problems.

**Comment by eshellman** [comment]
Thanks @tutlane , and welcome to free-programming-books!

**Pull request merged**